.changeset/add-mastra-discord.md

```
@@ -0,0 +1,28 @@
+---
+'@mastra/discord': minor
+---
+
+Added `@mastra/discord` for connecting Mastra agents to Discord. One Discord app serves many guilds, with Ed25519 verification of interaction requests, guild and global slash-command registration, and Gateway support for DMs and @mentions, and it ships a dual ESM + CJS build. Set `encryptionKey` or `MASTRA_ENCRYPTION_KEY` to encrypt the stored bot token at rest.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)